README.rst

```diff
@@ -91,6 +96,8 @@ Log in to 54.191.9.106 with the credentials from the ``auth`` section of the con
 
 The staging setup is missing the ability to trigger builds in response to pushes happening.
 
+The staging master will start Linux slaves on AWS EC2 automatically.  To start a Mac OS X slave, see below.
```

Comment:
Instead of "see below" it is better to link to a section - things may get moved around.